aries-site/src/pages/components/grid.mdx

```diff
@@ -13,3 +21,184 @@ import { GridExample } from '../../examples';
+content such as an [activity feed](/templates/dashboards#two-column-dashboard).
+
+Mixed-width columns are specified in an array. The number of items in the array dictate the number of columns 
+in the grid. 
+
+For example, columns specified as `columns={[ 'small', 'xsmall', 'xsmall' ]}` would create a three 
+column grid with corresponding widths: 
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/MixedColumns1.js"
+  width={{ min: 'medium' }}
+  height="auto"
+>
+  <MixedColumns1 />
+</Example>
+
+<br /><br />
+
+And, `columns={[ '2/3', '1/3' ]}` would result in two columns with proportional 
+widths:
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/MixedColumns2.js"
+  width={{ min: 'medium' }}
+  height="auto"
+>
+  <MixedColumns2 />
+</Example>
+
+
+#### Flexible Columns
+Flexible columns is where Grid really shines. They allow content to elegantly scale and designs to be dynamic.
+
+The main idea is defining columns as a min-max range. Between those bounds, the column is allowed to 
+flex according to its content 
+and the space available to the grid as a whole. This allows content to be consistently displayed in an 
+organized and cohesive fashion across a variety of screen widths.
+
+The min-max range is specified using a two-item array, instead of a single string. 
+The first item is the min and second max.
+
+```
+// 3 columns. First and second columns are defined as min-max 
+// ranges. The third, as a fixed width.
+
+columns={[
+  ['small', 'medium'], // column 1 is able to flex
+  ['medium', 'large'], // column 2 is able to flex
+  'small',             // column 3 is always 'small'
+]};
+```
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/MinMaxColumns1.js"
+  width={{ min: 'medium' }}
+  height="auto"
+>
+  <MinMaxColumns1 />
+</Example>
+
+<br/><br/>
+
+Using `auto` and/or `flex` as min-max values provides even more flexibility:
+
+ - `auto` allows the 
+column's width to be defined by its contents; it says, "make 
+my column as wide as my content needs."
+- `flex` allows the column to grow and fill the any excess width in the grid. If multiple columns use 
+`flex`, they share the excess width evenly.
+
+See `auto` and `flex` in action in the example below, or applied in a [dashboard template](/templates/dashboards#two-column-dashboard).
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/MinMaxColumns2.js"
+  width={{ min: 'medium' }}
+  height="auto"
+  showResponsiveControls={['fullscreen']}
+  overflow="auto"
+>
+  <MinMaxColumns2 />
```

Comment:
Curious what this would look like if they were both "auto"? Would it be valuable to have that included as well?

Reply:
I clarified some of the description above to describe that scenario. I'll push up the changes. Let me know if that satisfies your curiosity, or if an additional example is needed.

Reply:
This takes care of my curiosities. Think we can stick with that and no additional example is needed.